docs/whats_new.md

```diff
@@ -0,0 +1,89 @@
+# ROCm on Windows
+
```

Comment:
Make it a landing page for the following content:

- ROCm on Windows
- File Reorg
- Web site restructure (what you are doing now)
- And lots more to follow like backward compatibility